index.html

```diff
@@ -595,10 +597,10 @@ <h3>Proofs</h3>
 ibsa-2022, and jws-2022.
       </p>
 
-      <pre class="example" title="A DataIntegritySignature example using a NIST ECDSA 2022 Cryptosuite">
+      <pre class="example" title="A DataIntegrityProof example using a NIST ECDSA 2022 Cryptosuite">
 {
-  "@context": ["https://w3id.org/security/data-integrity/v1"],
-  "type": "DataIntegritySignature",
+  "@context": ["https://w3id.org/security/suites/data-integrity/v1"],
```

Comment:
The context URL: `https://w3id.org/security/data-integrity/v1` is actually correct (today, anyway).

Reply:
Fixed in f33680902c92e937fe7c629ac7263d011d576e7f.